modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -23,11 +23,11 @@ Previously, each Jenkins image supported only one version of {product-title} and
 
 .What additions are there with the OpenShift Jenkins and Jenkins Agent Base ImageStream and ImageStreamTag objects?
 
-By moving from an "in-payload" image stream to an image stream that references non-payload images, {product-title} can define additional image stream tags. Red Hat has created a series of new image stream tags to go along with the existing `"value": "jenkins:2"` and `"value": "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base:latest"` image stream tags present in {product-title} 4.10 and earlier. These new image stream tags address some requests to improve how the Jenkins-related image streams are maintained.
+By moving from an "in-payload" image stream to an image stream that references non-payload images, {product-title} can define additional image stream tags. Red Hat has created a series of new image stream tags to go along with the existing `"value": "jenkins:2"` and `"value": "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base-rhel8:latest"` image stream tags present in {product-title} 4.10 and earlier. These new image stream tags address some requests to improve how the Jenkins-related image streams are maintained.
```

Comment:
```suggestion
By moving from an in-payload image stream to an image stream that references non-payload images, {product-title} can define additional image stream tags. Red Hat has created a series of new image stream tags to go along with the existing `"value": "jenkins:2"` and `"value": "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base-rhel8:latest"` image stream tags present in {product-title} 4.10 and earlier. These new image stream tags address some requests to improve how the Jenkins-related image streams are maintained.
```
The use of the double-quotation marks here doesn't fall into any of the use cases cited in the ISG:

```

- Quotations.
- Words that are used in a special sense, but only if no other word clearly expresses the meaning. Do not overuse
- quotation marks for this purpose. Using a word in a special sense can cause problems for translation.
- Citations that have sentence-style capitalization and do not have any other highlighting.
- Hardware label names.
- Interface elements that have sentence-style capitalization and that need to be distinguished from surrounding text.

```

Reply:
Updated. Thanks for sharing the guideline, too.